common/types.go

```diff
@@ -353,10 +353,8 @@ func BigToAddress(b *big.Int) Address { return BytesToAddress(b.Bytes()) }
 // If s is larger than len(h), s will be cropped from the left.
 func HexToAddress(s string) Address { return BytesToAddress(FromHex(s)) }
 
-var relaxPrecompileRangeForTest bool
-
 // IsPrecompiledContractAddress returns true if the input address is in the range of precompiled contract addresses.
-func IsPrecompiledContractAddress(addr Address) bool {
+var IsPrecompiledContractAddress func(addr Address, rules interface{}) bool = func(addr Address, rules interface{}) bool {
```

Comment:
Could you add a comment saying `rules` are only for tests?

Reply:
I added it in https://github.com/kaiachain/kaia/pull/238/commits/1ab1826241fbc892b08ad80c261bf795ff424644. Thank you.